plugins/context_datalayer_condition.inc

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @file
+ * Context Datalayer condition plugin for Context API.
+ */
+
+/**
+ * Context Condition plugin to detect datalayer values.
+ */
+class context_datalayer_condition extends context_condition {
+  /**
+   * Implements options_form().
+   */
+  public function options_form($context) {
+    $form = parent::options_form($context);
+
+    $form['context_datalayer_condition']['data'] = array(
```

Comment:
At the start I had an annoying thing too, where when you hit "add" for condition, you'd actually get the reaction form being rendered via ajax...  I've literally NO IDEA why this happened ... It must be something within Context module that I couldn't be bothered to track down...